File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/cmd/entity/TerminalSNEntity.java
package com.shmedo.configlibrary.iot.cmd.entity;

import com.shmedo.configlibrary.ble.interfaces.Validater;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  11/19/20 <br/>
 * 描述：    终端设备 Sn 号码
 */
public class TerminalSNEntity implements Validater {
    private String sn;

    public TerminalSNEntity(String sn) {
        this.sn = sn;
    }

    @Override
    public void validate() {

    }

    @Override
    public String toString() {
        return "sn=" + sn;
    }
}

File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/cmd/entity/TerminalSNEntity.kt
package com.shmedo.configlibrary.iot.cmd.entity

import com.shmedo.configlibrary.ble.interfaces.Validater

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  11/19/20 <br></br>
 * 描述：    终端设备 Sn 号码
 */
class TerminalSNEntity(private val sn: String) : Validater {
    override fun validate() {}
    override fun toString(): String {
        return "sn=$sn"
    }
}